docs/user/tutorial.rst

```diff
@@ -304,16 +436,286 @@ attributes for reading and setting common headers, but you can always
 access any header by name with the ``req.get_header`` and ``resp.set_header``
 methods.
 
-Restart gunicorn, and then try sending a POST request to the resource
-(substituting test.jpg for a path to any JPEG you like.)
+With that explained, we can move onto making our service work.
+Edit ``app.py`` and pass in a path to the resource initializer.
+Right now, it can be the working directory that you've started the service from.
+
+.. code:: python
+
+    images = Resource(storage_path='.')
+
+Now you can run the tests again and see them pass!
+
+You can also restart Gunicorn, and then try sending a POST request to the resource
+yourself (substituting test.png for a path to any PNG you like.)
```

Comment:
I've just notices that in the text there's a lot of talk about PNGs, but in the shell examples there are JPGs. I should probably switch everything to PNG, right?